src/components/ScrollableContainer/ScrollableContainer.jsx

```diff
@@ -36,35 +36,69 @@ function ScrollableContainer({
     withSimpleBar,
   })
 
-  function renderBody() {
-    if (body) {
-      if (withSimpleBar) {
-        const { children, className, ...rest } = body.props
+  function renderSection(section) {
+    let sectionContent
+    let SectionUI
+    let sectionRef
+
+    if (section === 'header') {
```

Comment:
What do you think about extracting a function here to just return object with the 3 properties - should make it a bit cleaner and easier to understand :)

Reply:
I thought about it, but is only used once here, so I went with less indirection instead.